packages/web-lib/utils/format.bigNumber.ts

```diff
@@ -42,11 +33,10 @@ export function	bigNumberAsAmount(
 		symbolWithPrefix = ` ${symbol}`;
 	}
 
-	bnAmount = BN(bnAmount);
-	if (bnAmount.isZero()) {
+	if (bnAmount === Zero) {
 		return (`0${symbolWithPrefix}`);
 	}
-	if (bnAmount.eq(ethers.constants.MaxUint256)) {
+	if (bnAmount === 0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffn) {
```

Comment:
We should move this to the constants file -- although that's a bit weird and not at all readable, are you sure we need to use that and can't use something else for max uint?